### PR TITLE
Allow initiate_chat without passing message

### PR DESCRIPTION
**Pull request by bitnom**
A very minor change to call `self.get_human_input()` if `message` is not supplied.

- [x] I've included any doc changes needed for https://microsoft.github.io/autogen/. See https://microsoft.github.io/autogen/docs/Contribute#documentation to build and test documentation locally.
- [x] I've added tests (if relevant) corresponding to the changes introduced in this PR.
- [x] I've made sure all auto checks have passed.

### CODE REVIEW

**Comment by sonichi** [comment]
If folks are OK with this change, a test should be added to cover the added case.

**Review by rickyloynd-microsoft — APPROVED** [review]
One nice thing about supporting user-first chats is that it matches what users of ChatGPT are used to.

**Comment by sonichi** [comment]
@bitnom could you add a test to cover the new line added?

**Review by ekzhu — CHANGES_REQUESTED** [review]
Great PR! A test is needed to cover this case.

**Comment by bitnom** [comment]
> @bitnom could you add a test to cover the new line added?

it is committed

File: test/agentchat/test_human_input.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    def custom_a_get_human_input(prompt):
+        return "This is a test"
+
+    user_proxy.get_human_input = custom_a_get_human_input
+
+    user_proxy.register_reply([autogen.Agent, None], autogen.ConversableAgent.a_check_termination_and_human_reply)
+
+    user_proxy.initiate_chat(assistant, clear_history=True, message="Hello.")
+    # Test without supplying messages parameter
+    user_proxy.initiate_chat(assistant, clear_history=True)
```
Comment (ekzhu) [inline test/agentchat/test_human_input.py:41]: Perhaps an assert to check if the custom get human input function is called?

**Comment by bitnom** [comment]
> @bitnom could you add a test to cover the new line added?

should be good now

File: test/agentchat/test_human_input.py (hunk 1 above)
Comment (bitnom) [reply test/agentchat/test_human_input.py:39]: done

File: test/agentchat/test_async_get_human_input.py [hunk 2]
```diff
@@ -19,26 +21,27 @@
 @pytest.mark.skipif(skip, reason="openai not installed OR requested to skip")
 @pytest.mark.asyncio
 async def test_async_get_human_input():
+    try:
+        import openai
+    except ImportError:
+        return
```
Comment (sonichi) [inline test/agentchat/test_async_get_human_input.py:27]: not needed because of line 13-18

**Review by sonichi — COMMENTED** [review]
Minor suggestion.

File: test/agentchat/test_async_get_human_input.py (hunk 2 above)
Comment (sonichi) [inline test/agentchat/test_async_get_human_input.py:27]:
```suggestion
```

**Pull request merged**